src/CriticalTransitions.jl

```diff
@@ -50,6 +50,7 @@ export equilib, fixedpoints, basins, basinboundary, basboundary
 export simulate, relax
 export transition, transitions
 export fw_integrand, fw_action, om_action, action, geometric_action
+export div_drift
```

Comment:
To keep the namespace minimal, we should limit the amount of functions we export. I don't think we need to export `div_drift`

Reply:
I think it would be convenient to export because it is quite central to the Onsager-Machlup action. Of course one can compute it pretty easily. We could also consider not exporting `fw_integrand`

Reply:
But a user would never call it in a usual workflow right? If they want to call they can just import the function in the namespace or calll it with Criticaltransition.div_drift